polaris.dme

```diff
@@ -361,7 +361,6 @@
 #include "code\game\antagonist\station\renegade.dm"
 #include "code\game\antagonist\station\revolutionary.dm"
 #include "code\game\antagonist\station\rogue_ai.dm"
-#include "code\game\antagonist\station\thug.dm"

```

Comment:
Not entirely sure this is supposed to be removed(?)

Reply:
DAMMIT MERGE CONFLICT